hyperspy/tests/signal/test_2D_tools.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2007-2016 The HyperSpy developers
+#
+# This file is part of HyperSpy.
+#
+# HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with HyperSpy. If not, see <http://www.gnu.org/licenses/>.
+
+
+import numpy as np
+from scipy.misc import face
+from scipy.ndimage import fourier_shift
+
+import hyperspy.api as hs
+
+
+class TestSubPixelAlign:
+
+    def setUp(self):
+        ref_image = face()
+        center = np.array((256, 256))
+        shifts = np.array([(0.0, 0.0), (4.3, 2.13), (1.65, 3.58),
+                           (-2.3, 2.9), (5.2, -2.1), (2.7, 2.9),
+                           (5.0, 6.8), (-9.1, -9.5), (-9.0, -9.9),
+                           (-6.3, -9.2)])
+        s = hs.signals.Image(np.zeros((10, 100, 100)))
+        for i in xrange(10):
+            # Apply each sup-pixel shift using FFT and InverseFFT
+            offset_image = fourier_shift(np.fft.fftn(ref_image), shifts[i])
```

Comment:
This line is fine - it's just that I didn't realise face is an RGB image whereas lena was not... have switched to ascent to solve that in below.